pkg/ledger/ledger_monitor.go

```diff
@@ -80,29 +80,27 @@ func (m *Monitor) Start(ctx context.Context) {
 			}
 			// always instrument ledger latency even if ECS behavior is disabled.
 			stats.Set("reflector-ledger-latency", latency)
-			if !m.cfg.DisableECSBehavior {
-				switch {
-				case latency <= m.cfg.MaxHealthyLatency && (health == nil || *health != true):
-					// set a healthy attribute
-					if err := m.setHealthAttribute(ctx, m.cfg.HealthyAttributeValue); err != nil {
-						return errors.Wrap(err, "set healthy")
-					}
-					health = pointer.ToBool(true)
-				case latency > m.cfg.MaxHealthyLatency && (health == nil || *health != false):
-					// set an unhealthy attribute
-					if err := m.setHealthAttribute(ctx, m.cfg.UnhealthyAttributeValue); err != nil {
-						return errors.Wrap(err, "set unhealthy")
-					}
-					health = pointer.ToBool(false)
+			switch {
+			case latency <= m.cfg.MaxHealthyLatency && (health == nil || *health != true):
+				// set a healthy attribute
+				if err := m.setHealthAttribute(ctx, m.cfg.HealthyAttributeValue); err != nil {
```

Comment:
can you verify this doesn't involve the ecs api?

Reply:
I double checked and I do found `setHealthAttribute` will trigger [initiating a ECS client session](https://github.com/segmentio/ctlstore/blob/master/pkg/ledger/ledger_monitor.go#L138), given that do you still this PR make sense? If not I will just close it.